### PR TITLE
Build VPN platform pages 

### PR DESCRIPTION
**Pull request by maxxcrawford**
## Description

Create the following 6 pages: 
- [x] Windows 10
- [x] Mac
- [x] Linux
- [x] Mobile
- [x] iOS
- [ ] ~Android~

Other tasks: 

- [x] Decouple shared button from landing page

Notes: 
- The Android VPN platform page will be added in a later PR. 

## Issue / Bugzilla link

- #10011
- #10010
- #10009
- ~#10008~ Blocked on copy right now 
- #10007 
- #10006

## Testing

Test the following URLs (including subnav, pricing links) 
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/desktop/linux/
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/desktop/mac/
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/desktop/windows/
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/mobile/
- ~https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/mobile/android/~
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/mobile/ios/

### CODE REVIEW

File: bedrock/products/templates/products/vpn/platforms/linux.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import vpn_nav_platform_cta, vpn_platform_content_well with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% set vpn_countries = 30 %}
+{% set vpn_devices = 5 %}
+{% set vpn_devices_minus_1 = 4 %}
```
Comment (maxxcrawford) [inline bedrock/products/templates/products/vpn/platforms/linux.html:11]: The section of copy that uses this variable is:

> Take an extra measure of security when you’re online — both at home and out and about — with Mozilla VPN. For a small monthly fee, it uses advanced WireGuard® protocol to encrypt your internet connection on your Linux-based computer and **four other devices**, letting you stream shows, play games, shop, and go about your daily life online knowing that you’re secure.

I have a comment with the copywriting team about making this the maximum number. It also occurs in the iOS page, and needs fixing.

File: bedrock/products/urls.py [hunk 2]
```diff
@@ -20,8 +20,14 @@
         # Pages that do not use allowed_countries or default_monthly_price contexts
         page('vpn/desktop', 'products/vpn/platforms/desktop.html',
              ftl_files=['products/vpn/platforms/shared', 'products/vpn/platforms/desktop']),
-    )
-else:
-    urlpatterns += (
-        redirect(r'^vpn/desktop(/.*)?', 'products.vpn.landing', permanent=False),
```
Comment (maxxcrawford) [inline bedrock/products/urls.py:26]: Per @alexgibson, this can be removed as these are new pages.

File: l10n/en/brands.ftl [hunk 3]
```diff
@@ -137,6 +137,7 @@
 -brand-name-android = Android
 -brand-name-ios = iOS
 -brand-name-linux = Linux
+-brand-name-ubuntu = Ubuntu
```
Comment (maxxcrawford) [inline l10n/en/brands.ftl:140]: This technically was only used in the Linux.ftl file, but this seems the correct place to place variables like this.

File: l10n/en/products/vpn/platforms/ios.ftl [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+vpn-ios-page-title = { -brand-name-mozilla-vpn } for { -brand-name-ios } ({ -brand-name-iphone } & { -brand-name-ipad })
+
+vpn-ios-page-description = Get fast, easy security on your { -brand-name-iphone } or { -brand-name-ipad } through { -brand-name-mozilla }, one of the most trusted names in technology.
+
+# Mobile strings
+vpn-ios-hero-headline = Protect yourself online with { -brand-name-mozilla-vpn } for { -brand-name-ios }
+
+# Variables:
+#   $devices (number) - number of available devices
+vpn-ios-ability = Feel safe from hackers and prying eyes when you’re online — both at home and out and about — with { -brand-name-mozilla-vpn }. For a low monthly fee, it uses advanced { -brand-name-wireguard }® protocol to encrypt personal data on your { -brand-name-iphone }, { -brand-name-ipad } and three more devices, letting you stream shows, play games, shop, and go about your daily life online knowing that you’re secure.
```
Comment (maxxcrawford) [inline l10n/en/products/vpn/platforms/ios.ftl:17]: Revise "three more devices" to be the maximum number of devices or some other fix.

**Review by maxxcrawford — COMMENTED** [review]
Self-review. Note that the Android page is currently blocked on copy, so it has been intentionally omitted for now.

File: l10n/en/products/vpn/platforms/mac.ftl [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+vpn-mac-privacy-headline = Privacy at your fingertips
+vpn-mac-privacy-copy = With { -brand-name-mozilla-vpn }, your internet connection is encrypted and your IP address is protected from hackers and nosy internet providers thanks to { -brand-name-wireguard }® advanced protocol.
+
+vpn-mac-servers-headline = Access servers around the world
+
+# Variables:
+#   $countries (number) - number of available countries
+#   $servers (number) - number of available servers
+vpn-mac-servers-copy = Set your { -brand-name-mac-short }’s location to your choice of more than { $servers }+ servers in { $countries } countries. This lets you check the news, shop, stream and surf the web from just about anywhere.
```
Comment (peiying2) [inline l10n/en/products/vpn/platforms/mac.ftl:31]: should we add a `+` in in `{ $countries }+ countries` to stay consistent with Linux platform?

File: l10n/en/products/vpn/platforms/mac.ftl [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+# Variables:
+#   $devices (number) - number of available devices
+vpn-mac-devices-headline = Protect { $devices } devices with one subscription
+vpn-mac-devices-copy = Your computer, table and smartphone all deserve to be secure. Use { -brand-name-mozilla-vpn } to protect up to { $devices } { -brand-name-mac-short } and { -brand-name-ios } devices (in addition to { -brand-name-windows }, { -brand-name-linux } and { -brand-name-android }).
```
Comment (peiying2) [inline l10n/en/products/vpn/platforms/mac.ftl:36]: `table` -> `tablet`?

File: l10n/en/products/vpn/platforms/mac.ftl (hunk 5 above)
Comment (maxxcrawford) [reply l10n/en/products/vpn/platforms/mac.ftl:31]: I think so. There are a few other instances, but that was as approved in the copy doc.

File: bedrock/products/templates/products/vpn/platforms/ios.html [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+{% block page_title_full %}{{ ftl('vpn-ios-page-title') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block page_desc %}{{ ftl('vpn-ios-page-description') }}{% endblock %}
+
+{% block body_id %}mozilla-vpn-platforms-ios{% endblock %}
+
+{% set _utm_source = 'www.mozilla.org-vpn-ios-page' %}
+{% set _utm_campaign = 'vpn-product-page' %}
```
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/platforms/ios.html:21]: Should utm campaign be something specific for these pages?

File: bedrock/products/urls.py [hunk 8]
```diff
@@ -20,8 +19,14 @@
...
+        page('vpn/desktop/linux', 'products/vpn/platforms/linux.html',
+             ftl_files=['products/vpn/platforms/shared', 'products/vpn/platforms/linux']),
```
Comment (alexgibson) [inline bedrock/products/urls.py:23]: We should probably reorder these files as the first one is used for activation (same applies below)
```suggestion
             ftl_files=['products/vpn/platforms/linux', 'products/vpn/platforms/shared']),
```

**Review by alexgibson — CHANGES_REQUESTED** [review]
A couple of nits but otherwise these pages look good to me r+wc

I'll defer to Peiying on reviewing the Fluent strings.

File: bedrock/products/templates/products/vpn/platforms/ios.html (hunk 7 above)
Comment (maxxcrawford) [reply bedrock/products/templates/products/vpn/platforms/ios.html:21]: The `_utm_source` above is.

Comment (maxxcrawford) [reply bedrock/products/templates/products/vpn/platforms/ios.html:21]: @rraue Do you think I should change all of these to `vpn-platform-page`?

File: bedrock/products/urls.py (hunk 8 above)
Comment (maxxcrawford) [reply bedrock/products/urls.py:23]: Done!

**Review by alexgibson — APPROVED** [review]

**Pull request merged**